### DISCUSSION TITLE
出码报错

### DISCUSSION THREAD

**Discussion by w990752800** [opentiny/tiny-engine#405] [Q&A]
![image](https://github.com/opentiny/tiny-engine/assets/23731304/9748e171-fdd9-4222-9f0e-aa6db8e4a8dd)
![image](https://github.com/opentiny/tiny-engine/assets/23731304/07dc24ee-a859-48a8-81a6-9223e54a9058)

**Comment by w990752800** [comment]
使用的最新版的谷歌浏览器

**Comment by w990752800** [comment]
我使用 mockServer 就可以正常下载， 现在换成 data-center, 这种后端服务后，就报这个错了

**Comment by wenmine** [comment]
#366 已经收敛在issue中，本discussion关闭
